ruby_basic/taras.turchenko/homework_1/task_1.rb

```diff
@@ -0,0 +1,27 @@
+TIMEZONES = {
+    'America/Anchorage': '-09:00',
+    'Europe/London': '+00:00',
+    'Europe/Paris': '+01:00',
+    'Europe/Kyiv': '+02:00',
+    'Europe/Minsk': '+03:00'
+}
```

Comment:
'Freeze mutable objects assigned to constants.'

Reply:
I fixed it